src/language/lexer.ts

```diff
@@ -287,7 +287,7 @@ function unexpectedCharacterMessage(code: number): string {
 /**
  * Reads a comment token from the source file.
  *
- * #[\u0009\u0020-\uFFFF]*
+ * `#[\u0009\u0020-\uFFFF]*`
```

Comment:
It's a grammar so backtrick can be confusing here better to use code blocks for all grammar rules